packages/upload-client/test/index.test.js

```diff
@@ -453,6 +475,9 @@ describe('uploadDirectory', () => {
         onShardStored: (meta) => {
           carCID = meta.cid
         },
+        fetch: setupGetReceipt(() => {
+          return links[0]
```

Comment:
could we by the path from the url used in fetch figure out which one is the CID for this link instead?

Reply:
Oh yes that would be nicer, and allow it to respond appropriately when not found.

Reply:
sadly, I don't think we can as we would need to compute the task ID from the initial links. I think it's best I remove this altogether